docs/discover/search-for-relevance.asciidoc

```diff
@@ -19,13 +19,14 @@ Warsaw OR Venice OR Clear
 ```
 . If you don't see any results, expand the <<set-time-filter,time range>>, for example to *Last 7 days*.
 . From the list of *Available fields*, add `_score` and any other fields you want to the document table.
-. To sort the `_score` column in descending order, hover over its header, and set
-the sort icon to
-image:images/downward-arrow.png[Downward pointing arrow to indicate descending sorting].
 +
-At this point, you're doing a multi-column sort: first by `Time`, and then by `_score`.
-. To turn off sorting for the `Time` field, hover over its header, and set the sort icon to
-image:images/double-arrow.png[Arrow on both ends of the icon indicates sorting is off].
+At this point, you're sorting by the`timestamp` field.
```

Comment:
qq: on top of this file:

_In *Discover*, open the {data-source} dropdown, and select that data you want to work with._

is it really **that** data?

Reply:
good catch